helpers/wrapper/reports/SECURITY.md

```diff
@@ -1,150 +1,80 @@
-# Security Report: Wrapper
+# Wrapper: Security Report
 
-Smart Contract Security Report by Team Fluidity (team[at]fluidity[dot]io) and Phil Daian (feedback[at]stableset[dot]com)
-Hash of master used for report: [ef5cff0613532d27ecedb332e222ae0a75079841](https://github.com/airswap/airswap-protocols/commit/ef5cff0613532d27ecedb332e222ae0a75079841)
+Security report by Team Fluidity (team[at]fluidity[dot]io). Smart contracts are a nascent space, and no security audit procedure has been perfected. We welcome any suggestions and comments on this report, its contents, our methodology, or potential gaps in coverage.
```

Comment:
Are we having a bounty? If so could mention that here, just with a link to where they can find more info on a bug bounty?

Reply:
Bounty will be published next week and can be added to these reports when ready